resources/.tsan-ignore.txt

```diff
@@ -9,4 +9,8 @@ race:shared_count
 
 # Boost is not built with tsan support, either
 race:boost::detail::thread_data_base
-race:thread_data
\ No newline at end of file
+race:thread_data
+
+# Hyrise is able to handle concurrent access to MVCC-data correctly.
+# However, the MvccDeletePluginSystemTest lead to several TSan warnings. (#1493)
```

Comment:
> Hyrise is able to handle concurrent access to MVCC-data correctly.

Even if we could be sure that this was true, it would be a really bad reason to suppress a warning. The comment should explain what the suspected data race looks like, why it is actually thread-safe and why tsan still reports a false positive. Maybe we should also extract `end_cids[row_id.chunk_offset] = cid;` into its own method so that we can suppress just that.

Reply:
A hacky first try, I agree. If we get TSan under control with this, I'll try to document it better.

Reply:
Nope, the idea is good